pandas/tests/groupby/test_timegrouper.py

```diff
@@ -102,7 +102,7 @@ def test_groupby_with_timegrouper(self):
                 index=date_range(
                     "20130901", "20131205", freq="5D", name="Date", inclusive="left"
                 ),
-            )
+            ).astype({"Buyer": object})
```

Comment:
column `'Buyer'` is originally set to `0`, and then some strings are set (thus upcasting to `object`). Might as well construct it as `object` from the start - furthermore, this is the `expected` dataframe, so it doesn't change what's being tested